__tests__/app.test.js

```diff
@@ -35,4 +35,34 @@ describe("GET /api/topics", () => {
                 expect(body.msg).toBe("path not found")
             }) 
     })
+})
+
+describe("GET /api/articles/:article_id", () => {
+    test("Responds with an article object, which should match a list of provided properties.", () => {
+        return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then((res) => {
+                const article = res.body.article;
+                const output = {
+                    author: "butter_bridge",
+                    title: "Living in the shadow of a great man",
+                    article_id: 1,
+                    body: "I find this existence challenging",
+                    topic: "mitch",
+                    created_at: "2020-07-09T20:11:00.000Z",
+                    votes: 100
+                }
+                expect(article).toBeInstanceOf(Object);
```

Comment:
You don't really need this assertion as anything that's not an object will fail the `toEqual` check below it. If you want to leave it in there for a clearer error message that's totally ok 👍